my-website/docusaurus.config.js

```diff
@@ -0,0 +1,87 @@
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').DocusaurusConfig} */
+module.exports = {
+  title: 'My Site',
+  tagline: 'Dinosaurs are cool',
```

Comment:
CLI framework

Reply:
Made the changes. If there are any other changes please let me know